src/dev/amdgpu/pm4_packet_processor.cc

```diff
@@ -384,7 +384,7 @@ PM4PacketProcessor::mapQueues(PM4Queue *q, PM4MapQueues *pkt)
                 "Mapping mqd from %p %p (vmid %d - last vmid %d).\n",
                 addr, pkt->mqdAddr, pkt->vmid, gpuDevice->lastVMID());
 
-        gpuDevice->mapDoorbellToVMID(pkt->doorbellOffset,
+        gpuDevice->mapDoorbellToVMID(pkt->doorbellOffset << 2,
```

Comment:
If possible, it would be great to document why << 2

Reply:
Added comment